src/back/GameLauncher.ts

```diff
@@ -105,7 +105,9 @@ export namespace GameLauncher {
     }
     // Launch game
     let proc: ChildProcess;
-    switch (opts.game.applicationPath) {
+    let appPath: string = getApplicationPath(opts.game.applicationPath, opts.execMappings, opts.native)
+
+    switch (appPath) {
```

Comment:
This switch should use the raw application path since the ``getApplicationPath`` function can change a value of ``:flash:`` into something different

Reply:
`getApplicationPath` only modifies the raw app path for non-Windows platforms. To get `:flash:` to work on Mac, I have the following entry in execs.json:

```json
    {
      "win32": "FPSoftware\\Flash\\flashplayer_32_sa.exe",
      "linux": "FPSoftware/Flash/linux/flashplayer32SA",
      "darwin": ":flash:"
    },
```

If the `execMappings` are not applied, the `default` case is selected and the Launcher tries to execute `FPSoftware\\Flash\\flashplayer_32_sa.exe` on Mac (and it looks like it'll do this for Linux as well).

Reply:
Oh yeah you're right, my bad